src/Redux/Categories/Categories.js

```diff
@@ -0,0 +1,14 @@
+const CHECK_STATUS = 'bookstore-cms/categories/CHECK_STATUS';
+const initialState = [];
+
+/* eslint-disable default-param-last */
```

Comment:
- [ ] Kindly remove this line  & solve the linter manually. As this is going to be mandatory in the next milestone.